installer.sh

```diff
@@ -9,81 +10,96 @@ PROJECT_NAME=gbw
 BUILD_DIR=$(pwd)/.build
 BUILD_TARGET=""
 INSTALL_DIR=$(pwd)/.dist
-PACKAGE_DIR=$(pwd)/.packages
+PACKAGES_DIR=$(pwd)/.packages
+
+BROWSER_OPTIONS="<chromium|webkit|mswebview2>"
+BROWSER_DEFAULT="chromium"
 
-MSWEBVIEW_NUGET_V=1.0.3351.48
-CHROMIUM_NUGET_V=138.0.17
-BROWSER_OPTIONS="<chromium|mswebview2>"
 SYS_OPTS=""
 
 PATH=$(cygpath -w /ucrt64/bin):$PATH
 export PATH
+
 source ./.scripts/util.sh
-# For Windows, check that we are running in a MSYS2 UCRT environment
-check_env
+source ./.scripts/packages.sh
 
+# Check if we are running in a MSYS2 UCRT environment for Windows
+if ! check_win_env; then
+    return 1;
+fi
 
 SELECT_BROWSER_HELP="\
 #  Selects the browser engine to use
 ## Options:
-#      chromium                                                 Use Chromium Embedded Framework (CEF)
-#      mswebview2                                               Use Microsoft WebView2 (Windows only)
+#      chromium                                Use Chromium Embedded Framework (CEF)
+#      webkit                                                             Use Webkit
+#      mswebview2                                     Use MS WebView2 (Windows only)
 ## Default:
 #      chromium
 ## Usage:
-#      select_browser $BROWSER_OPTIONS                          
-"
+#      select_browser $BROWSER_OPTIONS"
 select_browser() {
     if [ "$(is_help "$@")" = true ]; then
         print_help "$SELECT_BROWSER_HELP"
-        [ $PS1 ] && return 0 || exit 0
+        return 0
     fi
 
-    select_browser_caller "$1"
+    if [ -z "$1" ]; then 
+        BROWSER=$BROWSER_DEFAULT
+        echo "Using the default browser '$BROWSER'"
+        return 0
+    fi
 
+    if [ "$1" = "mswebview2" ]; then
+        BROWSER="$1"
+    elif [ "$1" = "chromium" ]; then
+        BROWSER="$1"
+    elif [ "$1" = "webkit" ]; then
+        BROWSER="$1"
+    else
+        echo "'$1' was not a valid browser option: $BROWSER_OPTIONS"
+        select_browser
+        return 0
+    fi
+
+    echo "'$BROWSER' browser selected"
 }
```

Comment:
This is now much simpler.
A function should ideally do only ONE thing, and this one should set `BROWSER`.

Before, it was also trying to set `SYS_OPS` which was kind of out of place and making things fragile.